### DISCUSSION TITLE
"App preview failed to load" / "There was an error loading your app"

### DISCUSSION THREAD

**Discussion by s-okal** [Budibase/budibase#12396] [Help] [answered]
Hi everyone,

Yesterday I deployed Budibase on a Google Kubernetes cluster and although I can access the interface and import data from a CSV, I am unable to modify or view the app itself. Here are some screenshots:
<img width="1271" alt="err1" src="https://github.com/Budibase/budibase/assets/145070456/1ddad6e6-6695-42a8-bad4-935a8e694af0">
<img width="1274" alt="err2" src="https://github.com/Budibase/budibase/assets/145070456/0967ef12-115a-4f8a-8237-2207af9c5acd">
I can see these errors within the browser console:
<img width="535" alt="image" src="https://github.com/Budibase/budibase/assets/145070456/ed215882-32ed-4b1e-addf-f4c3011244bf">
Here is my diagnostic info:
{
  "browser": {
    "language": "en-US",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36",
    "platform": "Win32",
    "vendor": "Google Inc."
  },
  "server": {
    "budibaseVersion": "2.12.11",
    "hosting": "kubernetes",
    "nodeVersion": "v18.18.2",
    "platform": "linux",
    "cpuArch": "x64",
    "cpuCores": 2,
    "cpuInfo": "Intel(R) Xeon(R) CPU @ 2.20GHz",
    "totalMemory": "3.8365859985351562GB",
    "uptime": "0 day(s), 15 hour(s), 53 minute(s)"
  }
}
I am not sure if this is a configuration issue but I found someone in [this thread](https://github.com/Budibase/budibase/discussions/12269) who seems to have the same issue while using Kubernetes.
Any and all suggestions are appreciated!

**Comment by melohagan** [comment, marked as answer]
Hey @s-okal 

There's a few issues with our helm chart at the moment which may be causing issues. Work is being done here: https://github.com/Budibase/budibase/pull/12416
